crates/client-api/src/auth.rs

```diff
@@ -112,7 +135,7 @@ impl TokenClaims {
         &self,
         signer: &impl TokenSigner,
         expiry: Option<Duration>,
-    ) -> Result<String, JwtError> {
+    ) -> Result<(SpacetimeIdentityClaims, String), JwtError> {
```

Comment:
Slap a comment on here describing the return value? I believe it is the tuple `(claims, token)`, where `claims` is the contents of `token`.

Reply:
It's also not obvious to me why this is useful. Can't the caller just clone out of `self` themselves?

Reply:
`self` only has a subset of the claims, since we are adding claims for expiry and issued at.